resources/views/dashboard/controllers/events/view.blade.php

```diff
@@ -87,15 +87,51 @@
                                         </td>
                                         @if(Auth::user()->isAbleTo('events'))
                                             <td>
-                                                <span data-toggle="modal" data-target="#addrequest{{ $r->id }}">
-                                                    <button type="button" class="btn btn-success btn-sm simple-tooltip" data-placement="top" data-toggle="tooltip" title="Assign Position"><i class="fas fa-check"></i></button>
-                                                </span>
-                                                &nbsp;
-                                                <a href="/dashboard/controllers/events/view/{{ $r->id }}/un-signup" class="btn btn-danger btn-sm simple-tooltip" data-toggle="tooltip" title="Delete Request"><i class="fas fa-times"></i></a>
+
+                                                <div class="btn-group" role="group" aria-label="Actions">
+
+                                                    <button data-toggle="modal" data-target="#addrequest{{ $r->id }}" type="button" class="btn btn-success btn-sm simple-tooltip" data-placement="top" title="Assign Position">
+                                                        <i class="fas fa-check fa-fw"></i>
+                                                    </button>
+                                                    <button onclick="window.location.href='/dashboard/controllers/events/view/{{ $r->id }}/un-signup'" class="btn btn-danger btn-sm simple-tooltip" title="Delete Request">
```

Comment:
This does work, but the standard practice on the website is to use an anchor here instead of button with JS. So we probably should leave this as an anchor to follow along with the rest of the code.

Reply:
Alright, I’ll change it back. I originally changed it to a button so it would line up with the rest of the button group - if you make it a link it doesn’t connect up properly, but I’m sure I can find a way to make the link do that too